katello/hooks/pre_commit/09-version_locking.rb

```diff
@@ -9,17 +9,13 @@
 end
 
 if lock_versions
-  unless system('command -v foreman-maintain > /dev/null')
-    fail_and_exit('Locking of package versions was requested but foreman-maintain is not installed')
-  end
-  unless system('foreman-maintain packages -h > /dev/null 2>&1')
+  unless foreman_maintain('packages -h')
     fail_and_exit('Locking of package versions was requested but foreman-maintain version installed does not support it')
```

Comment:
This doesn't work because `foreman_maintain()` will call `fail_and_exit` already.